server/svix-server/migrations/20220428190104_message_expiration.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+ALTER TABLE message ADD COLUMN expiration timestamp with time zone;
```

Comment:
There is no default value here and it's not nullable, so it's broken. You can probably define a default value of "now + 90 days" in SQL (server side calculated), or make it nullable.

Reply:
`expiration` is nullable unless I add a NOT NULL constraint, right?

However, assuming adding a default value of "now + 90 days" will add that default value for _existing_ rows before the migration, then that could be a clever workaround for the first update operation in the worker.

Reply:
Confirmed, Postgres will apply a column's default value to existing rows when creating the column. I'll hold for your approval on this, but I think that setting the default expiration eliminates the need for the business logic update migration in the worker.

Reply:
Yup, I agree.

My comment about the not-null was done in the wrong place, it was not null in the model code, which is what I meant. :)